lib/alchemy/tinymce.rb

```diff
@@ -14,8 +14,8 @@ module Tinymce
       menubar: false,
       statusbar: true,
       toolbar: [
-        'bold italic underline | strikethrough subscript superscript | numlist bullist indent outdent | removeformat | fullscreen',
-        'pastetext charmap hr | undo redo | alchemy_link unlink anchor | code'
+        'undo redo pastetext | alchemy_link unlink anchor hr subscript superscript charmap | code | fullscreen',
+        'styleselect bold italic underline strikethrough | alignleft aligncenter alignright alignjustify | numlist bullist indent outdent'
```

Comment:
This is personal preference and should be done in your app instead.

```rb
# config/initializers/alchemy.rb
Alchemy::Tinymce.init = {
  toolbar: [
    'undo redo'
  ]
}
```

Reply:
Okay I understand. I just thought the default setup showed barely any options. Being able to do more out of the box by default is nice.

Reply:
This really depends on how you want to build your site. But Alchemy is meant to give the editor the _least possible_ options to format text. All structural and styling related things are in the control of the developer. We are very successful with that approach. This is why people choose Alchemy over Wordpress or other "Page-Model-Body-Column" CMSs.

If you want to give full control over the layout and styling of a page inside a richtext editor why use Alchemy in the first place? Wordpress and friends are a better fit then.

And again, if you disagree (and that's totally fine) just change the Tinyme config in your local app. That's why we gave developers the control over the Tinymce editor.